migrations/20230130124611-add-payment-method-stripe-payment-id-unique-index.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface) {
+    await queryInterface.sequelize.query(`
+      CREATE UNIQUE INDEX "PaymentMethods__stripePaymentMethodId" ON "PaymentMethods"(("data"->>'stripePaymentMethodId'))
```

Comment:
```suggestion
      CREATE UNIQUE INDEX "PaymentMethods__stripePaymentMethodId" ON "PaymentMethods"(("data"#>>'{stripePaymentMethodId}'))
```

I'm not 100% sure, but I had some trouble in the past using the `->>` operator.
Sequelize always defaults to `#>>` and for some reason, it seems that the operator will impact the indexing.

It would be nice to verify this behavior locally using `EXPLAIN ANALYZE`.

Reply:
you're right!